Automattic/wp-calypso
# Stats: Update stats-download component

Part of the stats maintenance project, this branch is a quick clean-up and rename of the directory for the `download-csv` component.

__To Test__
- Open up a site stats page for a site with a good amount of traffic
- Click "View All" in a component footer, like Posts & Pages
- On the subsequent Summary Page, verify the download csv button at the bottom indeed downloads a csv file

__Further Improvements__
There are a few patterns in this file that could be further abstracted as they are re-used often in stats, just noting here for myself:

- `module-content-text`
- `ul.documentation`
